### PR TITLE
mold 1.7.0

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ZhongRuoyu** [comment]
As the built-in Mach-O tests are already included, I suggest that the ELF tests be run on Linux as well. That makes sure everything (including the `mold -run` test being replaced) works as expected.

File: Formula/mold.rb [hunk 1]
```diff
@@ -62,10 +62,13 @@ def install
...
     inreplace buildpath.glob("test/elf/*.sh") do |s|
-      s.gsub!(%r{(`pwd`/)?mold-wrapper}, lib/"mold/mold-wrapper", false)
+      s.gsub!(%r{(\./|`pwd`/)?mold-wrapper}, lib/"mold/mold-wrapper", false)
       s.gsub!(%r{(\.|`pwd`)/mold}, bin/"mold", false)
-      s.gsub!(/-B[^\s]+/, "-B#{libexec}/mold", false)
+      s.gsub!(/-B(\.|`pwd`)/, "-B#{libexec}/mold", false)
     end
+    # Fix permission of test file.
+    # Remove on next release as fix is already merged upstream.
+    chmod "+x", buildpath/"test/elf/section-order.sh"
```
Comment (carlocab) [inline Formula/mold.rb:71]: Let's wait until the next release to use this then.

Comment (ZhongRuoyu) [reply Formula/mold.rb]: Sure.

**Comment by ZhongRuoyu** [comment]
By the way, with the changes in https://github.com/rui314/mold/commit/0d7a5a7284df7e4268efdc65207f3da44b145057, the Linux bottle is no longer relocatable; the `-ULIBDIR` trick is no longer working.

We could do something like this to achieve the same, though:

```diff
diff --git a/config.h.in b/config.h.in
index 203b4846..edd59107 100644
--- a/config.h.in
+++ b/config.h.in
@@ -1,2 +1,2 @@
 #define MOLD_VERSION "@mold_VERSION@"
-#define MOLD_LIBDIR "@CMAKE_INSTALL_FULL_LIBDIR@"
+#define MOLD_LIBDIR ""
```

What do you think, @carlocab?

**Pull request closed without merging**